Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/actions/index.ts

```diff
@@ -41,4 +43,13 @@ const formatNotificationDate = (notificationDate: string) => {
         notificationDate = notificationDate.replace(',', '\xa0\xa0');
     }
     return notificationDate;
+}
+
+const formatNotificationSendingDuration = (SendingStartedDate: string, SentDate: string) => {
+    let sendingDuration: string = "";
+    if (SendingStartedDate && SentDate) {
+        let timeDifference: number = new Date(SentDate).valueOf() - new Date(SendingStartedDate).valueOf();
```

Comment:
Should/can these valueOf()s be getTime()?

Also, I don't think you need ": number".

Reply:
done